main.go

```diff
@@ -108,7 +113,12 @@ func main() {
 		templates.SetProxiesURL(proxiesURL)
 	}
 
-	go utils.CheckCountry([]string{"Ukraine"})
+	countries := strings.Split(countryList, ",")
```

Comment:
Since this is a blocking code, can we only do it if strict mode or country list are specified? (I think it should be 'or' rather than 'and' since use case of having strict mode with no country is also valid to check the connection

Reply:
The reason I didn't do "or" is because country list does have default value currently, so it would be a somewhat breaking change.

Check out https://github.com/Arriven/db1000n/pull/302/commits/640d02300ab4ec9db18b4f8aced354d826bb9e25?w=1